Update blob-storage readme (#466)
logzio/logz-docs: _source/logzio_collections/_log-sources/azure-blob.md

| Parameter | Description |
|---|---|
| Resource group | Click **Create new**. Give a meaningful **Name**, such as "logziobBlobStorageIntegration", and then click **OK**. |
| Location | Choose the same region as the Azure services that will stream data to this Blob Storage. |
{:.paramlist}

###### In the SETTINGS section

| Parameter | Description |
|---|---|
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up [here]({{site.baseurl}}/user-guide/accounts/account-region.html). |
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to. |
| Format (Default: text) | Choose one of the supported parsing formats: json/csv/text  |
{:.paramlist}

> At the bottom of the page, agree to the terms and conditions. 
> and then click **Purchase** to deploy.
> 
> Deployment can take a few minutes.

Reviewer: ```suggestion
At the bottom of the page, agree to the terms and conditions. 
Then click **Purchase** to deploy. Deployment can take a few minutes.
```

| Parameter | Description |
|---|---|
| Resource group | Click **Create new**. Give a meaningful **Name**, such as "logziobBlobStorageIntegration", and then click **OK**. |
| Location | Choose the same region as the Azure services that will stream data to this Blob Storage. |
{:.paramlist}

###### In the SETTINGS section

| Parameter | Description |
|---|---|

> | Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up 
>  [here]({{site.baseurl}}/user-guide/accounts/account-region.html). |
> | Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to. |

Reviewer: ```suggestion
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up [here]({{site.baseurl}}/user-guide/accounts/account-region.html). |
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to. |
```

|---|---|

> | Logs listener host | Use the listener URL for your logs account region. For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html). |

Reviewer: ```suggestion
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up 
 [here]({{site.baseurl}}/user-guide/accounts/account-region.html). |
```

> | Logs account token | Use the [token](https://app.logz.io/#/dashboard/settings/general) of the logs account you want to ship to. |

Reviewer: ```suggestion
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to. |
```

| Format (Default: text) | Choose one of the supported parsing formats: json/csv/text  |
{:.paramlist}

> At the bottom of the page, select **I agree to the terms and conditions stated above**,

Reviewer: ```suggestion
At the bottom of the page, agree to the terms and conditions. 
```

| Parameter | Description |
|---|---|
| Resource group | Choose your existing resource group, and then click **OK**. |
| Location | Choose the same region as the Azure services that will stream data to this Blob Storage. |
{:.paramlist}

###### In the SETTINGS section

| Parameter | Description |
|---|---|
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up [here]({{site.baseurl}}/user-guide/accounts/account-region.html).  |
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to.  |
| Format (Default: text) | Choose one of the supported parsing formats: json/csv/text  |
{:.paramlist}

> At the bottom of the page, agree to the terms and conditions. 
> and then click **Purchase** to deploy.
> 
> Deployment can take a few minutes.

Reviewer: ```suggestion
At the bottom of the page, agree to the terms and conditions. 
Then click **Purchase** to deploy. Deployment can take a few minutes.
```

| Parameter | Description |
|---|---|
| Resource group | Choose your existing resource group, and then click **OK**. |
| Location | Choose the same region as the Azure services that will stream data to this Blob Storage. |
{:.paramlist}

###### In the SETTINGS section

| Parameter | Description |
|---|---|

> | Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up 
>  [here]({{site.baseurl}}/user-guide/accounts/account-region.html).  |
> | Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to.  |

Reviewer: ```suggestion
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up [here]({{site.baseurl}}/user-guide/accounts/account-region.html).  |
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to.  |
```

|---|---|

> | Logs listener host | Use the listener URL for your logs account region. For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html). |

Reviewer: ```suggestion
| Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up 
 [here]({{site.baseurl}}/user-guide/accounts/account-region.html).  |
```

> | Logs account token | Use the [token](https://app.logz.io/#/dashboard/settings/general) of the logs account you want to ship to. |

Reviewer: ```suggestion
| Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to.  |
```

| Format (Default: text) | Choose one of the supported parsing formats: json/csv/text  |
{:.paramlist}

> At the bottom of the page, select **I agree to the terms and conditions stated above**,

Reviewer: ```suggestion
At the bottom of the page, agree to the terms and conditions. 
```

##### Build an event subscription

> In the left menu of your storage account, select **Events**.

Reviewer: ```suggestion
Open your storage account. In the left menu, select **Events**.
```

##### Build an event subscription

> In your storage account, in the left menu, click **Events**.

Reviewer: ```suggestion
In the left menu of your storage account, select **Events**.
```
